DEVELOPMENT.md

```diff
@@ -79,3 +79,13 @@ The above commands can be repeated to create more shuttle nodes.
 ### API
 
 At this point you may begin adding files to estuary via the API described in the [documentation](https://docs.estuary.tech/api-content-add), or using the golang client [creek](https://github.com/iand/creek).
+
+## Building
+
+Estuary is an open source experimental IPFS node and is open for anyone to propose changes and modify. 
```

Comment:
Can you clarify what purpose this serves? I'm not sure.

Reply:
Given that estuary is open source, I figured we should indicate that anyone in the community can propose changes to it.